web/share/js/kvm/mouse.js

```diff
@@ -243,37 +249,43 @@ export function Mouse(__getGeometry, __recordWsEvent) {
 			});
 		}
 	};
-
 	var __streamWheelHandler = function(event) {
 		// https://learn.javascript.ru/mousewheel
 		// https://stackoverflow.com/a/24595588
 
 		event.preventDefault();
-
+		//set default rate of -5, but allow localStorage hid.mouse.scroll_rate value to be used.
+		var rate=-5;
+		try{
+			rate=-window.localStorage.getItem("hid.mouse.scroll_rate");
+		} catch (ReferenceError){
+			// no rate was d
+		}
 		if (!__absolute && !__isRelativeCaptured()) {
 			return;
 		}
 
 		let delta = {"x": 0, "y": 0};
-		if (tools.browser.is_firefox && !tools.browser.is_mac) {
+		//This is for firefox and chrome, but not on mac.
```

Comment:
Why?

Reply:
I have no idea.  I can verify Chrome and Firefox.  I can't verify mac, so I added a comment to make it apparent in case someone wants to fix it in the future or wants to know why things are working that way.  I'm not sure how it will work on Mac and mac handling was special to start with.

Reply:
Let's keep it as is

Reply:
Should I disable the slider on mac?  
`if (ismac) $( "#hid-mouse-scroll-slider" ).prop( "disabled", true );` in init?

Reply:
No, we have mac handler in else branch